samples/e2eTestUtils/jest-puppeteer-utils/serverUtils.js

```diff
@@ -3,29 +3,87 @@
  * Licensed under the MIT License.
  */
 
-const { exec } = require('child_process');
-const waitOn  = require('wait-on');
+const { spawn } = require('child_process');
+const http = require("http");
```

Comment:
https?  Are there localhost ssl related issues?

Reply:
All of our samples are hosted on http